src/contracts/core/DelegationManager.sol

```diff
@@ -434,7 +434,7 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
     }
 
     /**
-     * @notice Called by an avs to register an operator with the avs.
+     * @notice Called by the AVS's service manager contract to register an operator with the avs.
```

Comment:
if its called by the service manager, do we really need to pass the address of the service manager as a parameter?

Reply:
We don't pass the address in as a parameter?

Reply:
ehhh my bad I confused 2 functions.